src/script/script.cpp

```diff
@@ -269,7 +269,7 @@ bool CScript::HasValidOps() const
     while (it < end()) {
         opcodetype opcode;
         std::vector<unsigned char> item;
-        if (!GetOp(it, opcode, item) || opcode > MAX_OPCODE || item.size() > MAX_SCRIPT_ELEMENT_SIZE) {
+        if (!GetOp(it, opcode, item) || opcode > MAX_OPCODE || item.size() > (size_t)MAX_SCRIPT_ELEMENT_SIZE) {
```

Comment:
`MAX_SCRIPT_ELEMENT_SIZE` is  of type `unsigned`, so I don't see why this change is here.

Reply:
Sorry, should have been `(unsigned int)opcode`. Now updated. Please re-review :-)